src/solvers/prop/prop_conv.h

```diff
@@ -72,51 +72,52 @@ class prop_conv_solvert:public prop_convt
 public:
   prop_conv_solvert(const namespacet &_ns, propt &_prop):
     prop_convt(_ns),
-    use_cache(true),
-    equality_propagation(true),
-    freeze_all(false),
-    post_processing_done(false),
     prop(_prop) { }
 
-  virtual ~prop_conv_solvert() { }
+  virtual ~prop_conv_solvert() = default;
 
   // overloading from decision_proceduret
-  virtual void set_to(const exprt &expr, bool value) override;
-  virtual decision_proceduret::resultt dec_solve() override;
-  virtual void print_assignment(std::ostream &out) const override;
-  virtual std::string decision_procedure_text() const override
+  void set_to(const exprt &expr, bool value) override;
+  decision_proceduret::resultt dec_solve() override;
+  void print_assignment(std::ostream &out) const override;
+  std::string decision_procedure_text() const override
   { return "propositional reduction"; }
-  virtual exprt get(const exprt &expr) const override;
+  exprt get(const exprt &expr) const override;
 
   // overloading from prop_convt
   using prop_convt::set_frozen;
   virtual tvt l_get(literalt a) const override { return prop.l_get(a); }
-  virtual void set_frozen(literalt a) override { prop.set_frozen(a); }
-  virtual void set_assumptions(const bvt &_assumptions) override
+  void set_frozen(literalt a) override
+  {
+    prop.set_frozen(a);
+  }
+  void set_assumptions(const bvt &_assumptions) override
```

Comment:
the formatting of these one-line functions is inconsistent

Reply:
Also unnecessary IMHO.